### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

**Comment by lepiaf** [comment]
this PR depend on update of https://github.com/swarrot/swarrot/pull/224

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
```
Comment (odolbeau) [inline composer.json:14]:
```suggestion
        "php": ">7.2.5",
```

File: DependencyInjection/SwarrotExtension.php [hunk 2]
```diff
@@ -95,6 +95,7 @@ public function buildCommand(ContainerBuilder $container, string $name, array $c
             ->addTag('console.command', [
                 'command' => 'swarrot:consume:'.$name,
             ])
+            ->setPublic(true)
```
Comment (lepiaf) [inline DependencyInjection/SwarrotExtension.php:98]: I had an error on unit test Tests/DependencyInjection/SwarrotExtensionTest.php

I don't know if I should remove assert or set service public. Let me know.

File: composer.json (hunk 1 above)
Comment (odolbeau) [reply composer.json:14]: Up

File: DependencyInjection/SwarrotExtension.php (hunk 2 above)
Comment (odolbeau) [reply DependencyInjection/SwarrotExtension.php:98]: As the service as public before, let's keep it like this. :ok_hand:

**Review by odolbeau — APPROVED** [review]

**Pull request merged**